lib/IRGen/GenFunc.cpp

```diff
@@ -2603,7 +2603,7 @@ llvm::Function *IRGenFunction::getOrCreateResumePrjFn(bool forPrologue) {
         auto callerContext = IGF.emitAsyncResumeProjectContext(addr);
         Builder.CreateRet(callerContext);
       },
-      false /*isNoInline*/, forPrologue));
+      false /*isNoInline*/, true/*for prologue*/));
```

Comment:
Should we remove the `forPrologue` parameter from this method?

Reply:
this function is used in a ton of places, most of them with "for prologue" == false.

What it would be nice is to rename this parameter to something more meaningful like "skip debug info", since this is what this parameter does. The original patch mixed the motivation from call sites with the effect inside this function.

Reply:
Ah sorry I missed that this could be called for one of two async resume functions, and it uses `forPrologue` to pick the name.